packages/td-bank-integration/src/td-online-mart.ts

```diff
@@ -3,7 +3,7 @@ export interface TdOnlineMartHostWindow extends Window {
 }
 
 export interface TDCustomCheckoutSDK {
-    create(fieldType: FieldType): TdOnlineMartElement;
+    create(fieldType: FieldType, options?: object): TdOnlineMartElement;
```

Comment:
```suggestion
    create(fieldType: FieldType, options?: FieldOptions): TdOnlineMartElement;
```
lets create additional interface for this